composer.json

```diff
@@ -156,6 +156,10 @@
         {
             "type": "path",
             "url": "src/Symfony/Contracts"
+        },
+        {
+            "type": "path",
+            "url": "src/Symfony/Bridge/PhpUnit"
```

Comment:
Totally unsure about this, I don't see how we'll get unit tests to pass without the update on the bridge. Maybe that this deserves a second pull request?